3.13/README.md

```diff
@@ -1,58 +1,89 @@
-# Our plan for 3.13
+# Our plan for Python 3.13
 
-Author: Mark Shannon
+Author: Mark Shannon, Michael Droettboom
 
-### Overview 
+### Overview
 
-The plan for 3.13 is similar to early plans for 3.12.
-[3.12 plan](../3.12/README.md)
+The plan for 3.13 is similar to [early plans for 3.12](../3.12/README.md).
 
 The big difference is that we have now finished the foundational work that we need:
-* PEP 669 is implemented.
+
+* [Low impact monitoring (PEP 669)](https://peps.python.org/pep-0669/) is implemented.
 * The bytecode compiler is a much better state.
-* The interpreter generator is working
-* Experiments on the register machine are complete
-* We have a viable apporach to create a low-overhead maintainable machine code generator
+* The interpreter generator is working.
+* Experiments on the register machine are complete.
+* We have a viable approach to create a low-overhead maintainable machine code generator, based on [copy-and-patch](https://fredrikbk.com/publications/copy-and-patch.pdf).
 
-We can work on the tier 2 optimizer chain unimpeded.
+We plan four parallelizable pieces of work for 3.13:
 
-We plan two main pieces of work for 3.13:
 * The tier 2 optimizer
-* Faster memory allocation and deallocation, and reduced cycle GC ovehead 
+* Copy-and-patch machine code generator
+* Enabling subinterpreters from Python code ([PEP 554](https://peps.python.org/pep-0554/)).
+* Memory management
 
 ### The tier 2 optimizer
 
 See [Tier 2 optimizer](https://github.com/faster-cpython/ideas/issues/557) for an explanation of what the tier2 optimizer is.
 
 The workplan is roughly as follows:
-* Get the tier 2 interpreter working 
+* Get the tier 2 interpreter working
 * Generate (poor quality) superblocks
 * Implement basic superblock management
 * In parallel:
   * Add support for deoptimization to superblocks
   * Enhance the superblock creation code
   * Implement the specializer
   * Implement the partial evaluator
-  * Implement the copy-and-patch machine code generator
 
 Our goal for 3.13 is to reduce the time spent in the interpreter by at least 50%.
 
-[Detailed plan][tier2_plan.md]
+[Detailed plan](https://github.com/faster-cpython/ideas/issues/587).
+
+### Copy-and-patch machine code generator

```

Comment:
This belongs in the detailed plan for the tier 2 optimizer.
The machine code generator is part of the optimizer.